en/peripherals/frsky_telemetry.md

```diff
@@ -34,14 +34,19 @@ Ready-made cables (which include the required adapters) are available from:
   
 ## PX4 Configuration
 
-The typical port used for FrSky telemetry is TELEM2. This can be enabled by [setting](../advanced_config/parameters.md) the [SYS_COMPANION](../advanced_config/parameter_reference.md#SYS_COMPANION) parameter to 10.
+[Configure the serial port](../peripherals/serial_configuration.md) on which FrSky will run using [TEL_FRSKY_CONFIG](../advanced_config/parameter_reference.md#TEL_FRSKY_CONFIG) (typically `TELEM 2` is used for FrSky telemetry). 
+There is no need to set the baud rate for the port, as this is configured by the driver.
 
+> **Note** If `TEL_FRSKY_CONFIG` is not available in *QGroundControl* then you will need to [add the driver to the firmware](#firmware).
+
+No further configuration is required; FrSky telemetry auto-starts when connected and detects D or S mode.
+
+<!--

```

Comment:
@bkueng This is update to the FrSky telmetry. 
I have commented this bit (pending deletion) to check - do Pixhawk 3 Pro and Pixracer set these ports up by default still (as it says)? If so we should probably add that info here and to the serial config page.

Reply:
For Pixracer, yes. For Pixhawk 3 Pro, TELEM4 is now a configurable UART (so FrSky does not run by default).